invenio_users_resources/services/permissions.py

```diff
@@ -10,26 +10,40 @@
 
 from invenio_records_permissions import BasePermissionPolicy
 from invenio_records_permissions.generators import (
+    AdminAction,
     AnyUser,
     AuthenticatedUser,
     SystemProcess,
 )
 
+from invenio_users_resources.permissions import user_management_action
+
 from .generators import IfPublicEmail, IfPublicUser, Self
 
+UserManager = AdminAction(user_management_action)
```

Comment:
Ended up naming it "AdminAction" so it can be re-used in other admin-related activities.